src/Utils.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+
+contract Utils {
+    uint256 internal constant UINT256_MAX =
+        115792089237316195423570985008687907853269984665640564039457584007913129639935;
+
+    function bound(uint256 x, uint256 min, uint256 max) internal virtual returns (uint256 result) {
+        require(min <= max, "Test bound(uint256,uint256,uint256): Max is less than min.");
+
+        uint256 size = max - min;
+
+        if (size == 0) {
+            result = min;
+        } else if (size == UINT256_MAX) {
+            result = x;
+        } else {
+            ++size; // make `max` inclusive
+            uint256 mod = x % size;
+            result = min + mod;
+        }
+    }
```

Comment:
> The only cost was the inability to use log_named_uint in bound anymore - though this should be fine? I don't see why users would want Forge Std to log the bounded value.

I think it's useful when a fuzz run reverts, because you can read what the value was, which helps with debugging. `bound` is an internal function, so Forge doesn't show the result in traces.

We can avoid defining an event with a workaround to log with `console` if the framework supports it. If it doesn't, nothing will break.

```suggestion
    function bound(uint256 x, uint256 min, uint256 max) internal virtual returns (uint256 result) {
        require(min <= max, "Test bound(uint256,uint256,uint256): Max is less than min.");

        uint256 size = max - min;

        if (size == 0) {
            result = min;
        } else if (size == UINT256_MAX) {
            result = x;
        } else {
            ++size; // make `max` inclusive
            uint256 mod = x % size;
            result = min + mod;
        }
        
        address(0x000000000000000000636F6e736F6c652e6c6f67).staticcall(
            abi.encodeWithSignature("log(string,uint)", "Bound result", result)
        );
    }
```

Reply:
Love this! In the future we might want to create a private function for this so we can avoid repeating the `address(0x0000...)` syntax, but it works very well for the purposes of this PR.